client/client_test.go

```diff
@@ -4034,3 +4154,123 @@ func checkSignatures(t *testing.T, targetSignatureData []TargetSignedStruct, exp
 type expectation struct {
 	role, target string
 }
+
+func TestNotaryRepository_initialize(t *testing.T) {
+	type fields struct {
+		baseDir        string
+		gun            data.GUN
+		baseURL        string
+		changelist     changelist.Changelist
+		cache          store.MetadataStore
+		remoteStore    store.RemoteStore
+		CryptoService  signed.CryptoService
+		tufRepo        *tuf.Repo
+		invalid        *tuf.Repo
+		roundTrip      http.RoundTripper
+		trustPinning   trustpinning.TrustPinConfig
+		LegacyVersions int
+	}
+	type args struct {
+		rootKeyIDs         []string
+		rootCerts          []data.PublicKey
+		serverManagedRoles []data.RoleName
+	}
+	tests := []struct {
+		name    string
+		fields  fields
+		args    args
+		wantErr bool
+	}{
+	// TODO: Add test cases.
```

Comment:
@riyazdf good catch! I think it was generated by accident and checked into the code. I have gone over existing tests and they are not needed. :) Thanks!